django/core/management/commands/listurls.py

```diff
@@ -0,0 +1,277 @@
+import json
+from importlib import import_module
+from io import StringIO
+
+from django.conf import settings
+from django.contrib.admindocs.views import (
+    extract_views_from_urlpatterns,
+    simplify_regex,
+)
+from django.core.management import color
+from django.core.management.base import BaseCommand, CommandError, CommandParser
+from django.utils import termcolors
+
+FORMATS = (
+    "aligned",
+    "table",
+    "verbose",
+    "json",
+    "pretty-json",
+)
```

Comment:
I have added a comment here on the formats we are supporting: https://forum.djangoproject.com/t/should-the-show-urls-command-be-part-of-djangos-standard-library/20739/16 

The main thing would be having 1 format be approved as accessible from the accessibility team.